js/main.js

```diff
@@ -399,6 +399,8 @@ function initiate() {
             }
         }
 
+        /* Delete all comments */
+        latexString = latexString.replace(/%.*(?=(\n|$))/g, '');
```

Comment:
Good point, I should do something about escaped percent sign.
Yes, I can just write that, sorry, overcomplicated it.